wiki/Tournaments/OWC/2020/zh.md

```diff
@@ -115,36 +115,55 @@ osu! 2020 世界杯由 osu! team 和多位社区成员举办。
 | ![][flag_UY] | **乌拉圭** | **[daanit](https://osu.ppy.sh/users/6159669)**, [Madozito](https://osu.ppy.sh/users/4054429), [Saphity](https://osu.ppy.sh/users/5382216), [-PloX](https://osu.ppy.sh/users/6404583), [Kazuki K](https://osu.ppy.sh/users/6407282), [Rebo](https://osu.ppy.sh/users/6942259), [FDX](https://osu.ppy.sh/users/8987016) |
 | ![][flag_VN] | **越南** | **[Tuon](https://osu.ppy.sh/users/6673790)**, [Ui chan](https://osu.ppy.sh/users/5449433), [\_Kuroni\_](https://osu.ppy.sh/users/6159305), [Hoaq](https://osu.ppy.sh/users/7696512), [Huukun](https://osu.ppy.sh/users/8114571), [FukutoTojido](https://osu.ppy.sh/users/8266808), [sindes19](https://osu.ppy.sh/users/11021073) |
 
-## 比赛时间表：决赛
+## 比赛时间表：总决赛
 
-### 2020 年 11 月 28 日，星期六
+### 2020 年 12 月 6 日，星期日
 
 | A 队 | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |  |
 | --: | :-- | :-: | :-: | :-: | :-: |
-| 德国 ![][flag_DE] | ![][flag_AU] 澳大利亚 | **19:00 UTC+08** | 12:00 UTC+01 | 22:00 UTC+11 | ² |
-| 日本 ![][flag_JP] | ![][flag_BR] 巴西 | **22:30 UTC+08** | 23:30 UTC+09 | 11:30 UTC-03 | ² |
-
-### 2020 年 11 月 29 日，星期日
-
-| A 队 | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |  |
-| --: | :-- | :-: | :-: | :-: | :-: |
-| 美国 ![][flag_US] | ![][flag_CA] 加拿大 | **07:00 UTC+08** | 16:00 UTC-07 | 18:00 UTC-05 | ¹ |
-| 巴西 ![][flag_BR] | ![][flag_AU] 澳大利亚 | **09:00 UTC+08** | 22:00 (星期六) UTC-03 | 12:00 UTC+11 | ³ |
-| 日本 ![][flag_JP] | ![][flag_AU] 澳大利亚 | **18:00 UTC+08** | 19:00 UTC+09 | 21:00 UTC+11 | ³ |
-| 日本 ![][flag_JP] | ![][flag_DE] 德国 | **21:00 UTC+08** | 22:00 UTC+09 | 14:00 UTC+01 | ³ |
+| 加拿大 ![][flag_CA] | ![][flag_DE] 德国 | **02:00 UTC+08** | 13:00 (星期六) UTC-05 | 19:00 (星期六) UTC+01 | ¹ |
 
 ### 2020 年 11 月 30 日，星期一
```

Comment:
This date should be updated.

Reply:
Wait, what data? I'm not seeing the diff in en.

Reply:
![image](https://user-images.githubusercontent.com/19843905/100678118-bd478e80-336c-11eb-87f4-ed1516074e7e.png)

should  be 2020-12-06

Reply:
> ![image](https://user-images.githubusercontent.com/19843905/100678118-bd478e80-336c-11eb-87f4-ed1516074e7e.png)
> 
> 
> 
> should  be 2020-12-06

Ohh my bad

Reply:
#4547